src/constants/object.js

```diff
@@ -0,0 +1,38 @@
+export const ObjectTypes = {
```

Comment:
if we are going to add more schemas we might need to extend this. Should we be dynamically or maintain once we add more schemas ?

Reply:
`/schemas` in the backend will give you the current schemas